hsl-gl-map-v9-base.json

```diff
@@ -1916,7 +1916,7 @@
                 "line-join": "round"
             },
             "paint": {
-                "line-color": "#fff",
+                "line-color": "#ffffce",
```

Comment:
This doesn't look good. Maybe we should make new (stricter) style layer for cycle ways. When you zoom in Kamppi for example, the areas tagged with bicyle=yes are surrounded with yellow lines.

Reply:
I think #42 is the correct answer

Reply:
Well, I had no specs about how they should be different, just that they should be different. I'm good with #42, so I'll revert this part of the PR.